#70 Add FHIR data exchange sequence diagrams

ArdonToonstra opened the pull request:
Introduced two Mermaid sequence diagrams illustrating FHIR data exchange for individual resources and QuestionnaireResponse retrieval. Updated the data-exchange documentation to include these diagrams for better visualization of the request and response flows.

ArdonToonstra committed:
#70 Add FHIR data exchange sequence diagrams

Introduced two Mermaid sequence diagrams illustrating FHIR data exchange for individual resources and QuestionnaireResponse retrieval. Updated the data-exchange documentation to include these diagrams for better visualization of the request and response flows.

ArdonToonstra committed:
Fix include paths for Mermaid diagram markdown files

Updated the include statements to reference the correct '.md' file extensions for Mermaid diagrams in the data exchange documentation.

HarmkeKoning commented:
The diagrams look good! The box of the first diagram is quite small so that doesn't look really nice. I don't mind that it is placed inside a box but it would be nice of the box could be bigger. Right now it is really hard to get a good overview of the diagram.

ArdonToonstra committed:
Embed Mermaid diagram in HTML div

Replaced Markdown code block with a styled HTML div for the Mermaid sequence diagram to improve rendering and control over diagram display.

ArdonToonstra commented:
@HarmkeKoning  I tried again. What about this one? We can manually set the height in a <div>. Is this ok, or do we want it smaller/bigger?

MariekeMassa reviewed (approved):
Sequence diagrams look fine like this. Scroll bar is not ideal but the best (maintainable) solution for now.
